src/modules/Packages.rb

```diff
@@ -1814,24 +1728,19 @@ def Initialize_StageInitial(show_popup, base_url, log_url)
             return
           end
         end
-        if !CheckContentFile(initial_repository)
-          label = ContentFileProductLabel()
-          # bug #159754, release the mounted CD
-          Pkg.SourceReleaseAll
-          Pkg.SourceDelete(initial_repository)
-          initial_repository = nil
-          if !Popup.ContinueCancel(
-              # message popup, %1 is product name
-              Builtins.sformat(_("Insert %1 CD 1"), label)
-            )
-            @init_error = Builtins.sformat(_("%1 CD 1 not found"), label)
-            @init_in_progress = false
-            return
-          end
+
+        # bug #159754, release the mounted CD
```

Comment:
You have to remove the whole block, the condition checks whether the content file downloaded from medium is the same as the downloaded by linuxrc. If they are not the same the medium has been changed in the mean time.

Othwerwise the installation repository will be always removed and user repeatedly asked for CD1.

We do not have an equivalent check and I don't think it is a problem removing it.